noir-projects/noir-protocol-circuits/crates/types/src/abis/accumulated_data/combined_accumulated_data.nr

```diff
@@ -65,11 +72,18 @@ impl CombinedAccumulatedData {
             non_revertible.public_data_update_requests,
             revertible.public_data_update_requests
         );
-        assert_sorted_array(
+
+        // Just check a permutation here...
+        check_padded_permutation(
```

Comment:
We can just use `check_permutation` here because in `assert_deduped_array` we call `validate_array` on `deduped_array`, that makes sure it's padded.

Or we can keep this and call `array_length` on `deduped_array` in `assert_deduped_array`.

I would prefer the first cause that's less code to be tested and audited 😃